packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -631,6 +659,27 @@ class OAuthClient extends CozyStackClient {
       }
     )
   }
+
+  /**
+   * Check if the OAuth client'shas been revoked.
+   * If this is the case, call the onRevocationChange callback
+   *
+   * @async
+   * @returns {Promise<boolean>} A Promise that resolves to `false` if client is still valid, or `true` if it has been revoked.
+   * @throws {Error} If an error occurs while fetching the client information.
```

Comment:
With this implementation, this is not true, but we should throw https://github.com/cozy/cozy-client/pull/1372/files#r1274467375

Reply:
Alright, it's fine by me I think both throwing a generic error or the initial fetcherror is fine as long as some context info is added during handling of the initial error

Reply:
- [ ] Updated